src/hooks/useContractsFilter.js

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react'
+
+export const useContractsFilter = ({ data, filters }) => {
+  const [filteredJobs, setFilteredJobs] = useState([])
```

Comment:
Why `jobs`? Please pay attention, don't copy/paste blindly.

Reply:
My apologies for the oversight. forgot to adjust the naming. Thank you for catching that.